src/lib/plugins/addSortBy.ts

```diff
@@ -176,18 +178,17 @@ export const addSortBy =
 
 		const sortKeys = createSortKeysStore(initialSortKeys);
 		const preSortedRows = writable<BodyRow<Item>[]>([]);
-		const sortedRows = writable<BodyRow<Item>[]>([]);
+		//const sortedRows = writable<BodyRow<Item>[]>([]);
```

Comment:
Not sure why I left `sortedRows` here, but I appreciate the cleanup! You can go ahead and remove this entirely.